components/panel/color.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import PanelBody from './body';
+
+function PanelColor( { colorValue, title, initialOpen = false, ...props } ) {
+	return (
+		<PanelBody
+			{ ...props }
+			{ ...{ initialOpen } }
```

Comment:
Aside from being a bit more difficult to read, there is a potential (minor) performance impact here on iterating an additional object to extend the props of the element, vs. the more basic `initialOpen={ initialOpen }`

See:

http://babeljs.io/repl/#?babili=false&browsers=ie%2011&build=&builtIns=true&code_lz=MYewdgzgLgBADgJxHCMC8MDeMCGAuGARgBoYAjAgJhgF8BuAKFElmHSxmAIGZbGnw0GAFMANsIC2wsFADKiYTgAm7ABQMYMADwAxECA2aOAOlOJkqGoc3ZTx7Gxq1rMKAEso4tNgDaAXWdNAHoAPgYASn5mITFJaSgAORB5BEUVDHVNXX0XWzMkFECjYG9OIs13T2FS_yLQiLogA&debug=false&circleciRepo=&evaluate=false&lineWrap=false&presets=es2015%2Creact&targets=&version=6.26.0

Reply:
Thank you @aduth improvement applied!